category_theory/category.lean

```diff
@@ -21,7 +21,7 @@ import tactic.tidy
 
 namespace category_theory
 
-universes u v
+universes v u
```

Comment:
a curious choice...

Reply:
You should put a comment right here because it's a really subtle move

Reply:
Will do!

Reply:
What do you think of:

```
universes v u -- The order in this declaration matters: v often needs to be explicitly specified while u often can be omitted
```

Reply:
I'm also thinking of adding:

```lean
-- declare the `v`'s first; see `category_theory.category` for an explanation
```

next to the other `universes` declarations.